### PR TITLE
Added multilabel document task data reader

### PR DESCRIPTION
**Pull request by nicolasahar**
Added functionality to the DocumentClassificationDataReader class to support multilabel outputs. Assumes you have the label space numericalized but can easily be changed.

### CODE REVIEW

**Comment by JohnGiorgi** [comment]
@nicolasahar Can we write this in a multi-label agnostic way? E.g. it could accept single label and multi-label document classification tasks. If not we need to break into two readers, not overwrite the existing reader

File: drbert/data/dataset_readers.py [hunk 1]
```diff
@@ -94,3 +104,16 @@ class SequenceLabellingDatasetReader(DatasetReader):
 
 class NLIDatasetReader(DatasetReader):
     pass
+
+if __name__ == "__main__":
+   
+    # Params for Cohort Task
```
Comment (JohnGiorgi) [inline drbert/data/dataset_readers.py]: This looks like it was used locally for testing? We should remove it from the repo

File: drbert/constants.py [hunk 2]
```diff
@@ -7,6 +7,8 @@
 PHI = 'PHI'
 BERT_MAX_SENT_LEN = 512
 
+COHORT_DATA_DIR = None
```
Comment (JohnGiorgi) [inline drbert/constants.py]: This is not used as far as I can tell?

File: drbert/data/dataset_readers.py [hunk 3]
```diff
@@ -94,3 +104,16 @@ class SequenceLabellingDatasetReader(DatasetReader):
...
+    path = None
+    partitions = None
+    tokenizer = AutoTokenizer.from_pretrained('bert-base-uncased', do_lower_case=True)
+    label_vocab = COHORT_DISEASE_CONSTANTS
+    batch_sizes = None
+
+    ICD_iters = DocumentClassificationDatasetReader(path, partitions, tokenizer, label_vocab, batch_sizes=batch_sizes, lower_case=True)
+
+    print(ICD_iters.text_to_iterator())
```
Comment (JohnGiorgi) [inline drbert/data/dataset_readers.py]: This looks like it was used locally for testing? We should remove it from the repo

**Comment by nicolasahar** [comment]
Yea i added these params in for local testing … i will delete

**Comment by nicolasahar** [comment]
I will delete these too

**Comment by nicolasahar** [comment]
Yea it should be pretty straightforward to make it agnostic. i will add it in tonight

**Pull request merged**